specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/getComputeOneSkuUsages.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "resourceUrl": "management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus",
```

Comment:
>management.azure.com [](http://example.com/codeflow?start=20&length=20)

Requiring `management.azure.com` to be specified as a parameter in the SDK is not a natural SDK pattern and will cause user confusion.

Typically swaggers don't use x-ms-parameterized-host and instead declare a path parameter called {scope} that is specified in paths. See blueprint or authorization swaggers for examples.

Reply:
I will update it the examples.

Reply:
The Quota RP is using - ARM Extension model - https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderRegistrationRoutingTypes.html#extension
ARM team suggested to check Insights RP for extension model. So, I was using the pattern from timeseriesinsights.json

We can add {scope} -= If it works fine for the extension model. Please suggest?

 "/{scope}/providers/Microsoft.Authorization/roleAssignments/{roleAssignmentName}": 
And in example:
"scope": "/subscriptions/subId/resourcegroups/rgname"

Reply:
The ARM Management URL changes by cloud, so should we use following pattern? It will work for all the clouds-

 "x-ms-parameterized-host": {
    "hostTemplate": "{environmentFqdn}",
    "useSchemePrefix": true,
    "positionInOperation": "first",
    "parameters": [
      {
        "$ref": "#/parameters/EnvironmentFqdn"
      }
    ]
  },

Reply:
updated to {scope} as suggested.